builder.go

```diff
@@ -132,8 +116,7 @@ func (b *Bob) maybeQueue(path string) {
 		// we couldn't find the file
 		// most likely a deletion
 		delete(mtimes, path)
-		b.execute()
-		return
+		return true
 	}
 
 	mtime := stat.ModTime()
```

Comment:
Yes, the library still gets tons of events for things like swap files that we don't care about so this allows us to only trigger the build when the modified time has changed.